.tina/schemas/features/genericFeature.ts

```diff
@@ -0,0 +1,23 @@
+import { TinaTemplate } from "tinacms";
+import { baseFeatureFields } from ".";
+
+const importedGenericFeatureBlockSchema: TinaTemplate = {
+  name: "importedGenericFeature",
+  label: "Imported Generic Feature",
+  ui: {
+    defaultItem: {
+      title: "Title placeholder",
+      text: "Phasellus scelerisque, libero eu finibus rutrum, risus risus accumsan libero, nec molestie urna dui a leo.",
+    },
+  },
+  fields: [
+    ...baseFeatureFields,
```

Comment:
If this line is remove and I don't spread an imported object here, everything is fine. This worked on `@tinacms/cli@0.60.5`